Cataclysm/APLs/ArmsWarrior.simc

```diff
@@ -0,0 +1,49 @@
+Smufrik Arms warrior
+
+actions.precombat+=/berserker_stance
+actions.precombat+=/commanding_shout,if=assigned_shout.commanding_shout&buff.my_battle_shout.down&buff.my_commanding_shout.down
+actions.precombat+=/battle_shout,if=assigned_shout.battle_shout&buff.my_battle_shout.down&buff.my_commanding_shout.down
+
+actions+=/run_action_list,name=aoe,if=active_enemies>2
+actions+=/call_action_list,name=init
+actions+=/sunder_armor,if=variable.should_sunder&variable.emergency_sunder
+actions+=/pummel
+actions+=/run_action_list,name=berserker_stance,if=buff.berserker_stance.up
+actions+=/run_action_list,name=battle_stance,if=buff.battle_stance.up

```

Comment:
Should there be any kind of fallback list or actions if you're not in Battle or Berserker?  Even if it is defaulting to tell you to get into Battle Stance?  Otherwise, people assume there's an error when the addon has nothing to recommend.

Reply:
Aaah that might be the problem i noticed when I testet earlier on ill push an update for that. Good call👍